includes/Core/API/ApiRouter.php

```
@@ -0,0 +1,884 @@
+				array( 'status' => 500 )
+			);
+		}
+	}
+
+	/**
+	 * Create preset.
+	 *
+	 * @param \WP_REST_Request $request The REST API request.
+	 * @return \WP_REST_Response The response object.
+	 */
+	public function create_preset( \WP_REST_Request $request ): \WP_REST_Response {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)